pkg/florist/user.go

```diff
@@ -70,19 +71,31 @@ func SupplementaryGroups(username string, groups ...string) error {
 // UserSystemAdd adds the system user 'username' and group 'username', with
 // home directory 'homedir' and mode 0o755.
 func UserSystemAdd(username string, homedir string) error {
+	return UserSystemAddWithUID(username, homedir, -1)
+}
+
+// UserSystemAdd adds the system user 'username', group 'username' and force userid to a
+// given number, with home directory 'homedir' and mode 0o755.
```

Comment:
```
// UserSystemAddWithUID adds the system user 'username' with home directory 'homedir'
// and user ID 'uid'. In case the user ID is already taken, UserSystemAddWithUID will fail.
// The user group will be 'username' and the mode of the home directory will be 0o755.
// Normally this is not the right function to use: see [UserSystemAdd] instead.
```